docs/spec/spec.md

```diff
@@ -1,13 +1,45 @@
 Lets do https://github.com/knative/serving/blob/master/docs/spec/spec.md style.
 
-For:
+# Knative Source API Spec
 
- - JobSource
+## Resource YAML Definitions
 
-the params it takes.
-  what they mean.
+### JobSource
 
-- spec.template
-- spec.backoffLimit
-- spec.parallel
-- etc that are for k8s jobs...
\ No newline at end of file
+```yaml
+apiVersion: sources.knative.dev/v1alpha1
+kind: JobSource
+metadata:
+  name: my-jobsource
+  labels:
+    controller-tools.k8s.io: "1.0"
```

Comment:
what is this label?

Reply:
Honestly not sure, I think it's an artifact from some example I was looking at